config/webpack.config.js

```diff
@@ -234,7 +234,7 @@ module.exports = env => {
       hot: true
     };
     // Source maps
-    config.devtool = 'inline-source-map';
+    config.devtool = (process.env.DEVTOOL || 'inline-source-map');
```

Comment:
Some bikeshedding: should it be `WEBPACK_DEVTOOL`?

Reply:
let me end the bikeshedding ASAP, yes let's rename it to `WEBPACK_DEVTOOL` if possible and never talk about it again :-)